src/domUpdates.js

```diff
@@ -1,69 +1,182 @@
-import {validateLogin} from './scripts';
+import {validateLogin, hotel, updateBookings} from './scripts';
+import {fetchBookings} from './apiCalls';
+import Booking from './classes/Booking';
 
 const cardsSection = document.querySelector('.booking-cards-wrapper');
 const billSection = document.querySelector('.total-spent-wrapper');
 const loginButton = document.querySelector('.login-button');
 const username = document.getElementById('username');
 const password = document.getElementById('password');
 const main = document.querySelector('main');
-const customerDashboard = document.querySelector('.customer-dashboard');
+const customerDashboard = document.getElementById('customer-dashboard');
+const bookingDashboard = document.getElementById('booking-dashboard');
 const loginPage = document.querySelector('.login')
+const menu = document.querySelector('.menu');
+const bookMenuButton = document.getElementById('book-menu-button');
+const dashboardMenuButton = document.getElementById('dashboard-menu-button');
+const dateInput = document.getElementById('date-input');
+const availableRoomsSection = document.getElementById('availableRooms');
+const filterButton = document.querySelector('.filter-button');
+const typeFilters = document.querySelectorAll('.checkbox-js');
+
+const querySelectors = {
 
-const qs = {
-  username,
-  password
 }
 
 const domUpdates = {
-  displayBookings(hotel, bookings) {
+  populateBookings() {
```

Comment:
Does it seem okay to leave this as one function? Was thinking of maybe splitting it up but not sure if that might just complicated since it all seems to be for once purpose (updating the user's bookings that display on the page). I'm also planning on changing this function to sort the cards by date

Reply:
Suggested change: 
the if path executes: populateEmptyBookingsError() 
the else path executes: populateCurrentCustomerBookings()

Reply:
genius, thank you!